app/main.py

```diff
@@ -427,15 +468,22 @@ def change_map_style_main(map_style_button_input, alert_button_input, map_style_
 
 
 @app.callback(
-    [Output('img_url', 'children'),
-     Output('live_alert_header_btn', 'children'),
-     Output('live_alerts_marker', 'children')],
-    Input('interval-component', 'n_intervals'),
-    State('map_style_button', 'children')
+    [
+        Output("img_url", "children"),
+        Output("live_alert_header_btn", "children"),
+        Output("live_alerts_marker", "children"),
+    ],
+    Input("interval-component-homepage", "n_intervals"),
```

Comment:
If I am not mistaken, this means that we have two callbacks (this one and the one right above) being triggered by the same input right? Couldn't they be merged into a unique callback then? 

Or maybe if we want to keep a notion of order in the triggering of the two callbacks, maybe this second callback could be triggered by changes to the alert data storage component? The input could directly be:

`Input('div_live_alerts_data', 'data')`.

Reply:
I don't like to merge the two. They are doing very separate stuff: one is calling the API, the other is creating stuff on the map. The logic is very different. 

I'm proposing the following:
- `update_style_components_with_alert_metadata` with the input that you suggested